src/foraging_gui/Foraging.py

```diff
@@ -4237,9 +4239,9 @@ def _generate_upload_manifest(self,FIP=False):
                 's3_bucket':'private',
                 'processor_full_name': 'AIND Behavior Team',
                 'modalities':{
-                    'behavior':self.TrainingFolder.replace('\\','/'),
-                    'behavior-videos':self.VideoFolder.replace('\\','/'),
-                    'fib':self.PhotometryFolder.replace('\\','/')
+                    'behavior':[self.TrainingFolder.replace('\\','/')],
+                    'behavior-videos':[self.VideoFolder.replace('\\','/')],
+                    'fib':[self.PhotometryFolder.replace('\\','/')]
                     },
                 'schemas':[
                     os.path.join(self.MetadataFolder,'session.json').replace('\\','/'),
```

Comment:
Can we add the data description if detected?

Reply:
I'm not sure if the watchdog service uses the data description. The data description will still get uploaded. Lets make that a separate issue so we can merge this in today